common/metrics/interfaces.go

```diff
@@ -119,7 +116,7 @@ type (
 	// Reporter is an interface for base constructor for metrics client.
 	// Deprecated
 	Reporter interface {
-		NewClient(logger log.Logger, serviceIdx ServiceIdx) (Client, error)
+		MetricProvider() MetricProvider
 		Stop(logger log.Logger)
 		UserScope() UserScope
```

Comment:
Just to confirm, we plan to remove this in near future as well, right?

Reply:
confirmed.